packages/kauri-wallet/tslint.json

```diff
@@ -28,8 +28,7 @@
     "interface-over-type-literal": true,
     "label-position": true,
     "max-line-length": [
-      true,
-      140
+      false
     ],
     "member-access": false,
     "member-ordering": [
```

Comment:
On line 73 can we change :     

`"no-unused-expressions": true,`
to
`"no-unused-expressions": ["error", { "allowShortCircuit": true, "allowTernary": true }],`

So we can do short cuts like `data.transactions && data.transactions.forEach()`

Reply:
great suggestion, just updated 🔍